redislock.go

```diff
@@ -131,9 +165,14 @@ func Obtain(ctx context.Context, client RedisClient, key string, ttl time.Durati
 	return New(client).Obtain(ctx, key, ttl, opt)
 }
 
+// Obtain is a short-cut for New(...).ObtainMany(...).
+func ObtainMany(ctx context.Context, client RedisClient, keys []string, ttl time.Duration, opt *Options) (*Lock, error) {
+	return New(client).ObtainMany(ctx, keys, ttl, opt)
+}
+
 // Key returns the redis key used by the lock.
```

Comment:
Comment doesn't match the method name.

Doesn't prevent us from merging and then polishing it ourselves.

Reply:
That's a breaking change and most people will not use it. Two options:

1. Please keep the original `Key` method and return the first value.
2. Add a `MultiLock` type

Reply:
Ooops it look like this comment about the `Key` method is lost.
You made the comment on this code (which replaced an existing `Key` method.
```golang
// Key returns the redis key used by the lock.
func (l *Lock) Keys() []string {
	return l.keys
}
```
I updated the code like this in my last changes:
```golang
// Key returns the redis key used by the lock.
// If the lock hold multiple key, only the first is returned.
func (l *Lock) Key() string {
	return l.keys[0]
}

// Keys returns the redis keys used by the lock.
func (l *Lock) Keys() []string {
	return l.keys
}
```
Assuming users that use many locks will use Keys. And single lock users can use Key. No more breaking changes.